src/profile/components/deleteProfile/DeleteProfile.tsx

```diff
@@ -45,12 +46,12 @@ function DeleteProfile(): React.ReactElement {
       }
     },
     onError: error => {
-      if (checkBerthError(error.graphQLErrors)) {
-        setBerthError(true);
-      } else {
-        Sentry.captureException(error);
-        createToast({ type: 'error' });
-      }
+      const standardErrorMessage =
+        error.graphQLErrors && error.graphQLErrors.length
+          ? error.graphQLErrors[0].message
+          : error.message;
+      Sentry.captureException(new Error(standardErrorMessage));
```

Comment:
This sends only the first GraphQL error to Sentry. Why not send all of them, if there are many?

Reply:
Can there be many, if possible errors are:

- PROFILE_DOES_NOT_EXIST_ERROR
- MISSING_GDPR_API_TOKEN_ERROR
- CONNECTED_SERVICE_DELETION_NOT_ALLOWED_ERROR
- CONNECTED_SERVICE_DELETION_FAILED_ERROR

There is no specification for Sentry, so can't say what should be sent. Previous version did not sent graphQL errors at all.

But the DeleteProfileError.tsx assumes there can be multiple, so this should too.

Reply:
I think it always needs to be assumed that there can be multiple errors. Also the documented error code lists aren't necessarily exhaustive.